src/aiodynamo/client.py

```diff
@@ -315,61 +396,68 @@ def table(self, name: TableName) -> Table:
         key: Dict[str, Any],
         *,
         return_values: ReturnValues = ReturnValues.none,
-        condition: ConditionBase = None
+        condition: Condition = None,
     ) -> Union[None, Item]:
         key = py2dy(key)
         if not key:
             raise EmptyItem()
 
+        payload = {
+            "TableName": table,
+            "Key": key,
+            "ReturnValues": return_values.value,
+        }
+
         if condition:
-            (
-                condition_expression,
-                expression_attribute_names,
-                expression_attribute_values,
-            ) = ConditionExpressionBuilder().build_expression(condition)
-        else:
-            condition_expression = (
-                expression_attribute_names
-            ) = expression_attribute_values = None
-        resp = await self.core.delete_item(
-            **clean(
-                TableName=table,
-                Key=key,
-                ReturnValues=return_values.value,
-                ConditionExpression=condition_expression,
-                ExpressionAttributeNames=expression_attribute_names,
-                ExpressionAttributeValues=expression_attribute_values,
-            )
-        )
+            params = Parameters()
+            payload["ConditionExpression"] = condition.encode(params)
+            payload["ExpressionAttributeNames"] = params.get_expression_names()
+            payload["ExpressionAttributeValues"] = params.get_expression_values()
```

Comment:
I wonder if there should be a static member like `Parameters.blank_expression_values` 🤔

Reply:
I don't understand what you mean.